### DISCUSSION TITLE
Anyone know of a software freedom-protecting front-end for browsing GH itself?

### DISCUSSION THREAD

**Discussion by strypey** [mendel5/alternative-front-ends#179] [Ideas]
Now that a bunch of code in GitHub repos isn't visible without running MicroSoft's proprietary JS, it would be great to have a software freedom-protecting front-end for browsing GH itself. Of course, Free Code projects moving off GH onto community-hosted code forges is the ideal solution, but in the meantime...

**Comment by youtubevanced7** [comment]
I’ve been looking for something similar! It’s great to see discussions around software freedom when using platforms like GitHub. Most mainstream browsers and extensions either compromise privacy or rely on proprietary services, which is frustrating. A front-end that prioritizes user control, privacy[,](https://vancedyoutube.org/) and open-source principles would be a game-changer. It would be especially helpful for developers, researchers, and contributors who want to engage with repositories without tracking or data collection. If anyone knows of a project or tool that acts as a software freedom-friendly interface for browsing GH, please share! I’d love to try it out and contribute.

**Reply by ltguillaume** [reply]
https://codeberg.org/gothub/gothub
